src/Reflection/ReflectionObject.php

```diff
@@ -66,30 +64,6 @@ public static function export($instance = null) : string
     }
 
     /**
-     * Cannot instantiate this way, use ReflectionObject::createFromInstance
-     *
-     * @throws \LogicException
-     */
-    public static function createFromNode(
-        Reflector $reflector,
-        ClassLikeNode $node,
-        LocatedSource $locatedSource,
-        ?NamespaceNode $namespace = null
-    ) : ReflectionClass {
-        throw new LogicException('Cannot create a ReflectionObject from node - use ReflectionObject::createFromInstance');
-    }
-
-    /**
-     * Cannot instantiate this way, use ReflectionObject::createFromInstance
-     *
-     * @throws \LogicException
-     */
-    public static function createFromName(string $className) : ReflectionClass
```

Comment:
Note: dropped because it is not an instance method, so an override does not really serve any purpose. The user would get a `ReflectionClass` instance anyway.

Reply:
Also: not really a BC break since this method was only throwing